c7n/resources/glue.py

```diff
@@ -160,3 +160,52 @@ def process(self, resources):
                     self.log.error(
                         "Exception deleting glue dev endpoint \n %s",
                         f.exception())
+
+
+@resources.register('glue-job')
+class GlueJob(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'glue'
+        enum_spec = ('get_jobs', 'Jobs', None)
+        detail_spec = None
+        id = name = 'Name'
+        date = 'CreatedOn'
+        dimension = None
+        filter_name = None
+        arn = False
+
+    permissions = ('glue:GetJobs',)
+
+    @property
+    def generate_arn(self):
```

Comment:
fwiw these go away in the arn type resolver branch thats in a separate pr. no need to change anything here though.